### PR TITLE
chore: Release v0.19.0

### PR DESCRIPTION
**Pull request by lexnv**
### [Fixed]

- Fixed connections processing await on server shutdown  ([#1153](https://github.com/paritytech/jsonrpsee/pull/1153))
- fix: include error code in RpcLogger  ([#1135](https://github.com/paritytech/jsonrpsee/pull/1135))
- fix: downgrade more logs to `debug`  ([#1127](https://github.com/paritytech/jsonrpsee/pull/1127))
- fix(server): remove `MethodSinkPermit` to fix backpressure issue on concurrent subscriptions  ([#1126](https://github.com/paritytech/jsonrpsee/pull/1126))
- fix readme links  ([#1152](https://github.com/paritytech/jsonrpsee/pull/1152))

### [Changed]

- server: downgrade connection logs to debug  ([#1123](https://github.com/paritytech/jsonrpsee/pull/1123))
- refactor(server): make `Server::start` infallible and add `fn builder()`  ([#1137](https://github.com/paritytech/jsonrpsee/pull/1137))

### [Added]

- added uptest new rust lib  ([#1151](https://github.com/paritytech/jsonrpsee/pull/1151))
- Add subway  ([#1124](https://github.com/paritytech/jsonrpsee/pull/1124))
- Add Trin to users and sort list alphabetically  ([#1119](https://github.com/paritytech/jsonrpsee/pull/1119))

### CODE REVIEW

**Comment by jsdw** [comment]
I think the "make Server::start infallible and add fn builder()" is a breaking change isn't it, so we can't just bump the patch version here right?

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,12 +4,33 @@ The format is based on [Keep a Changelog].
...
+
+### [Changed]
+
+- server: downgrade connection logs to debug  ([#1123](https://github.com/paritytech/jsonrpsee/pull/1123))
+- refactor(server): make `Server::start` infallible and add `fn builder()`  ([#1137](https://github.com/paritytech/jsonrpsee/pull/1137))
+
+### [Added]
+
+- added uptest new rust lib  ([#1151](https://github.com/paritytech/jsonrpsee/pull/1151))
+- Add subway  ([#1124](https://github.com/paritytech/jsonrpsee/pull/1124))
```
Comment (jsdw) [inline CHANGELOG.md:25]: Nit: this isn't really important (and should prolly be "Add Subway to users list"

**Review by jsdw — APPROVED** [review]
LGTM!

**Comment by jsdw** [comment]
> [Revert "tests: Change expected error to accomodate rust from gitlab"](https://github.com/paritytech/jsonrpsee/pull/1155/commits/b54b390da616323888e08365d0046680b4083cdc)

Hehe I suspected this might be the case; same happened in another recent PR :) Best to work with latest rust and we can ignore gitlab for now until it's updated if need be!

**Comment by lexnv** [comment]
Yep exactly! Wanted to confirm that the commit [tests: Change expected error to accomodate rust from gitlab](https://github.com/paritytech/jsonrpsee/pull/1155/commits/ac705591295d1e9c7f27768491b622da7595d571) will solve the issue for the gitlab runner, that to fallback to our github one.

That was just to make sure we can continue with this PR while progress is being made on the CI/CD side

File: CHANGELOG.md (hunk 1 above)
Comment (lexnv) [reply CHANGELOG.md:25]: Yep that makes sense, will remove this since I'd place it with the same priority as `chore` PRs 👍

**Review by tadeohepperle — APPROVED** [review]

**Comment by niklasad1** [comment]
> fix(server): remove MethodSinkPermit to fix backpressure issue on concurrent subscriptions (https://github.com/paritytech/jsonrpsee/pull/1126)

is also a breaking change so good call making this v0.19

**Review by niklasad1 — APPROVED** [review]

**Review by Mmagkooo — APPROVED** [review]

**Pull request merged**